PrestaShop/PrestaShop
# Migrate "Configure > Advanced Parameters > Webservices" - part 1 (configuration form)

Linked to https://github.com/PrestaShop/prestafony-project/issues/37

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop |
| Description?  | Migrate BO page 'webservice' to Symfony using modern controllers and form handlers - part 1 |
| Type?         | new feature |
| Category?     | BO |
| BC breaks?    | yes |
| Deprecations? | no |
| Fixed ticket? |  |
| How to test?  | See below |

### How to test

Login on backoffice, then browse `/admin-dev/index.php/configure/advanced/webservice/`. This shows the SF version of the form of the legacy page `/admin-dev/index.php?controller=AdminWebservice`. Only the form "configuration", not the listing (this will be delivered later).

You can check the 2 forms behave in the same way.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/9310)
